h2o-r/h2o-package/R/models.R

```diff
@@ -2730,9 +2736,17 @@ h2o.varimp_plot <- function(model, num_of_features = NULL){
 h2o.std_coef_plot <- function(model, num_of_features = NULL){
   # check that the model is a glm
   if(model@algorithm[1] != "glm") stop("Warning: model must be a GLM")
-
+  maxcoeff = 1
+  if (model@model$model_summary["family"]=="multinomial") {
+    coeff_table <- model@model$standardized_coefficient_magnitudes
+    sorted_table <- coeff_table[order(abs(coeff_table$coefficients)),]
+    norm_coef <- sorted_table$coefficients
+    sort_norm <- norm_coef
+    maxcoeff = max(norm_coef)
+  } else {
```

Comment:
ensured that the standard coefficient plot of multinomial is grabbing the correct field. It is standardized-coefficient_magnitude and not the coefficients_table